docs/docs/coverage/kubernetes.md

```diff
@@ -0,0 +1,21 @@
+# Kubernetes
+
+When scanning a Kubernetes cluster, Trivy differentiates between the following:
+
+1. Cluster infrastructure (e.g api-server, kubelet, addons)
+1. Cluster configuration (e.g Roles, ClusterRoles). 
+1. Application workloads (e.g nginx, postgresql).
+
+When scanning any of the above, Trivy differentiates between the Kubernetes Resource definition (i.e the YAML that defines it), and the container image if relevant.
```

Comment:
this is a bit unclear to me -- the way I understand it
> Whenever Trivy scans either of these Kubernetes resources, the container image is scanned separately to the Kubernetes Resource Definition (the YAML manifest) that defines the resource.

However that does not explain "if relevant" -- when is it relevant vs when is it not?

What do you think?

Reply:
"if relevant" because not all resource definitions has an image. for example, a service resource doesn't have an image.

Reply:
the way you wrote it sounds better I'll change, thanks